src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -360,6 +368,12 @@ export const WorkspaceMenuTrigger = ({ children, canShare, isOwner, setCloningWo
         tooltipSide: 'left',
         onClick: () => setSharingWorkspace(true)
       }, [makeMenuIcon('share'), 'Share']),
+      h(MenuButton, {
+        disabled: !isOwner,
+        tooltip: !isOwner && 'You have not been granted permission to toggle the lock for this workspace',
```

Comment:
Since the rest of the UI refers to this with action as "lock" or "unlock", I think it would be preferable to use the same terms here instead of "toggle the lock".

Reply:
Resolved in 06484da3bef1355fc5207a4ec0ba48742a0df32e